relay-server/src/services/autoscaling.rs

```diff
@@ -21,15 +21,15 @@ pub struct AutoscalingMetricService {
     /// This will always report `1` unless the instance is shutting down.
     up: u8,
     /// Gives access to AsyncPool metrics.
-    async_pool: EnvelopeProcessorServicePool,
+    async_pool: Option<EnvelopeProcessorServicePool>,
 }
 
 impl AutoscalingMetricService {
     pub fn new(
         memory_stat: MemoryStat,
         envelope_buffer: PartitionedEnvelopeBuffer,
         handle: Handle,
-        async_pool: EnvelopeProcessorServicePool,
+        async_pool: Option<EnvelopeProcessorServicePool>,
```

Comment:
Mh, we could also disable the service and not run it at all in Proxy mode.

Reply:
Could be of value to people running the proxy relay instance, no? Or would we point them to statsd metrics directly instead?